frontend/src/components/molecules/ListHeader/index.tsx

```diff
@@ -3,15 +3,29 @@ import './style.css';
 
 type Props = {
   title: string;
+  h1?: boolean;
 };
 
-export const ListHeader: React.FC<Props> = ({ title, children = null }) => (
-  <div className="list-header-container">
-    <div className="list-header fr-my-3w fr-mx-auto">
-      <span className="fr-m-0 fr-text--lead">{title}</span>
-      {children}
+export const ListHeader: React.FC<Props> = ({
+  title,
+  h1 = false,
+  children = null,
+}) => {
+  const getTitle = () => {
+    if (h1) {
```

Comment:
I think we better have an homogenous header everywhere to limit the amount of code.